site/layouts/community/single.html

```diff
@@ -0,0 +1,6 @@
+{{ define "main" }}
+<article class="markdown">
+  <h1 class="mb-8">{{ .Title }}</h1>
+  {{- .Content -}}
+</article>
+{{ end }}
```

Comment:
missing newline

Reply:
There's no newline in the posts layouts ( which this is a copy of ) - do you want to add them to both layouts ? I'm assuming you are thinking after the </article> ?